plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,44 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
+function OPDSBrowser:jumpToPage(viewer, count)
+    local input_dialog
+    input_dialog = InputDialog:new{
+        title = _("Enter Page Number"),
+        input = "",
+        input_type = "number",
+        input_hint = "Page # ".."(" .. "1 - " .. count .. ")",
```

Comment:
```suggestion
        input_hint = "(" .. "1 - " .. count .. ")",
```

Reply:
Oh, that's what you meant. Sorry!

Reply:
As written it would've put the untranslated string "Page #" on the screen, but it's not really necessary to put "Page number/#" everywhere you can regardless. So I think this way it's a win/win.

For future reference, the correct way to write this if we wanted it localized would be:
```lua
T(_("Page # (1 - %1)"), count)
```
But like I said, I don't want to see Page # here regardless. ^_^

For more info see:
http://koreader.rocks/doc/modules/ffi.util.html#template
http://koreader.rocks/doc/modules/gettext.html